aiida_vasp/calcs/tests/test_vasp.py

```diff
@@ -26,24 +26,24 @@ def test_write_potcar(vasp_calc_and_ref):
     """Check that POTCAR is written correctly."""
     vasp_calc, _ = vasp_calc_and_ref
 
-    with managed_temp_file() as temp_file:
-        vasp_calc.write_potcar(temp_file)
-        with open(temp_file, 'r') as potcar_fo:
+    with managed_temp_object() as temp_object:
+        vasp_calc.write_potcar(temp_object)
+        with open(temp_object, 'r') as potcar_fo:
             result_potcar = potcar_fo.read()
         assert 'In_sv' in result_potcar
         assert 'As' in result_potcar
         assert 'In_d' in result_potcar
         assert result_potcar.count('End of Dataset') == 2
 
         if isinstance(vasp_calc.inputs.structure, get_data_class('structure')):
-            multipotcar = MultiPotcarIo.read(temp_file)
+            multipotcar = MultiPotcarIo.read(temp_object)
             potcar_order = [potcar.node.full_name for potcar in multipotcar.potcars]
             assert potcar_order == ['In_sv', 'As', 'In_d', 'As']
 
 
 @ONLY_ONE_CALC
 def test_write_chgcar(localhost_dir, vasp_calc, vasp_inputs, vasp_chgcar):
-    """Test that CHGAR file is written correctly."""
+    """Test that CHGAR object is written correctly."""
```

Comment:
How about 'CHGCAR object'-> 'CHGCAR file'.

Reply:
Agreed, when I continue with this today I will look into this in more detail.